plurmy/orchestrate.py

```
@@ -226,10 +226,13 @@ def process(args, processing_queue_name: str, working_queue_name: str):
     Submit jobs to a Slurm controller using sbatch directives to process data
     from a given processing queue.
 
-    This func handles atomically pushing from the processing queue to the working
+    This func handles automically pushing from the processing queue to the working
     queue and then popping messages off the working queue upon sucessful 
     completion.
 
+    This function requires a PLURMY_CONDASH environment variable that points to 
```

If this function requires the env variable, we should check for it and raise if it is not set. The doc string should then not mention that the env variable is optional.